firefox-ios/Client/Frontend/TrackingProtection/TrackingProtectionHeaderView.swift

```diff
@@ -46,7 +45,9 @@ final class TrackingProtectionHeaderView: UIView, ThemeApplicable {
     }
 
     private var closeButton: CloseButton = .build { button in
-        button.layer.cornerRadius = 0.5 * UX.closeButtonSize
+        let viewModel = CloseButtonViewModel(a11yLabel: "",
+                                             a11yIdentifier: "Test Identifier")
```

Comment:
We should use real values for the a11y label and identifier.

Reply:
Yes sorry i don't know why i forgot it there